### ISSUE TITLE
Notification of 'new' reply even though there is none

### ISSUE THREAD

**Issue by dmp1ce** [PeerioTechnologies/peerio-client#190]
For some of my threads I get notifications for new replies even though there are none.  I click on the thread and the 'new' notification goes away but will often show up again, but I don't know the reason why.

**Comment by skylarnagao** [comment]
Are you experiencing this still with version 1.0.4.2?

**Comment by dmp1ce** [comment]
How do I tell what version I am on?

**Comment by anri-asaturov** [comment]
@dmp1ce when you start Peerio, before you log in you can see version number in the bottom left corner.

**Comment by dmp1ce** [comment]
Yes, I am on 1.0.4.2 and this is still occurring for me.

**Comment by flohdot** [comment]
@dmp1ce The expected behaviour for the green "modified" marker on conversations is that it appears when there is a new (unread) reply in the conversation, as well as when your conversation partners receive your messages and send read receipts. Are you seeing green markers even when no new read receipts are received?

**Comment by dmp1ce** [comment]
I receive this error twice from the person I was communicating with "Your contact attempted to send a message, but it could not be properly received by your account." That was on "2 Mar 2015 3:14PM"

We were able successfully send some messages after that but now I get the notifications of a new message even today, but I have not received or sent a message since "2 Mar 2015 6:07PM"

**Comment by dmp1ce** [comment]
I am communicating with the same person on several other threads and I do not have this problem on those. Only this one thread keeps notifying me. I haven't been able to determine when it decides to notify me but it isn't all the time. At least every day it notifies me.

**Comment by anri-asaturov** [comment]
@dmp1ce thanks for your help with fixing this problem. May I ask you to go to this problematic thread, expand all the messages in it by clicking on "expand" button and scroll through all the messages like you are reading them very fast. It would be very helpful to know if the error has gone or still there after this.

**Comment by dmp1ce** [comment]
@anri-asaturov I don't see an "expand" button. Where should it be? I did scroll through all of the messages though.

**Comment by anri-asaturov** [comment]
If you have not more than 10 messages in the thread then there will be no "expand" button, it's ok. When you have more than 10 messages the button will appear on top of your conversation. Thanks!

**Comment by dmp1ce** [comment]
I think I have less than 10 messages. I noticed that the notification time stamp is "2 Mar 2015 3:14PM" which is the same time as those errors. I just received the notification again.

**Comment by anri-asaturov** [comment]
Great, thanks for your help, we'll investigate this further.

**Comment by flohdot** [comment]
Ah, the problem is probably that some sort of decryption error means a read receipt was never sent, leaving the message marked as unread forever. Changing the issue name to reflect this. 

Another quick question @dmp1ce: Did you at any point remove (and presumably re-add) this contact?

**Comment by dmp1ce** [comment]
No, I never removed and re-added the contact. I believe the contact was trying out the Chrome and Mac client though. I'm not sure if he was switching clients around the same time. I was using Chrome the entire time.

**Comment by flohdot** [comment]
@dmp1ce Okay, thanks a lot!

**Comment by partyp** [comment]
I'm the one who is messaging @dmp1ce . I have mainly been using the Mac client, although I did switch to the Chrome version a few weeks back because I was having issues receiving any messages at all on the Mac client. The Mac client seems to be working fine for me now though. Im using Version 1.0.4.2.
